modules/acquia_telemetry/src/Event.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\acquia_telemetry;
+
+use JsonSerializable;
+
+/**
+ * An event object as defined by Amplitude API specifications.
+ *
+ * @see https://developers.amplitude.com/#http-api
+ */
+class Event implements JsonSerializable {
```

Comment:
Let's make this class `final`, and all the internal (currently protected) things `private`. We want to draw strong boundaries around this class. Core doesn't allow private right now, but because we are Lightning we can happily ignore that policy. :)